v20.2/build-a-typescript-app-with-cockroachdb.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Build a TypeScript App with CockroachDB and TypeORM
+summary: Learn how to use CockroachDB with the TypeORM framework.
+toc: true
+twitter: false
+---
+
+This tutorial shows you how run a simple application built with [TypeORM](https://typeorm.io/#/).
+
+## Step 1. Start CockroachDB
+
+{% include {{page.version.version}}/app/start-cockroachdb.md %}
+
+## Step 2. Create a database
+
+{% include {{page.version.version}}/app/create-a-database.md %}
+
+### Step 3. Get the code
+
+Clone [the code's GitHub repository](https://github.com/typeorm/cockroachdb-example).
+
+### Step 4. Update the connection parameters
+
+Open the `ormconfig.ts` file, and edit the ORM configuration parameters:
+
+<section class="filter-content" markdown="1" data-scope="local">
+
+- Replace the value for `port` with the port to your cluster.
+- Replace the value for `username` with the user you created earlier.
+- Replace the value for `password` with the password you created for your user.
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="cockroachcloud">
+
+- At the top of the file, uncomment the `import * as fs from "fs";` line.
```

Comment:
Which file? That line isn't in `ormconfig.json` and I don't see it any other place.